types/applications/application.d.ts

```diff
@@ -450,31 +448,31 @@ declare namespace Application {
      * A named "base application" which generates an additional hook
      * @defaultValue `null`
      */
-    baseApplication: string;
+    baseApplication: string | null;
```

Comment:
Is `null` here (and for the other properties)? I would have expected all of them to be optional (oh well, maybe not even that because the constructor of `Application` takes a `Partial<Application.Options>`), but why would `null` be a valid value?

Reply:
That is because of what the `defaultOptions` getter returns. That's what they are initially set as, if that methods is not overridden.

Reply:
OK, makes sense